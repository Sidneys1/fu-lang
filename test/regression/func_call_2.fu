test2: u8() => 22;

test: u8() => test2() + 20;

main: u8() => test();
