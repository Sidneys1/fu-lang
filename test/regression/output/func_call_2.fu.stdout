---------- test\regression\func_call_2.fu ----------
[2m   3 | test: u8() => [0;2mtest2() + 2[2m0;[0m[0;0m <-- Checks for infix operator '+' are not implemented! [0;2m(test\regression\func_call_2.fu:3:15-25)
[0m[m